frontend/src/metabase/dashboard/actions/cards.js

```diff
@@ -237,3 +242,73 @@ export const addActionToDashboard =
       }),
     );
   };
+
+export const autoApplyParametersToNewCard =
```

Comment:
This action seems to be unused. Since we have another similar function for that in `metabase/frontend/src/metabase/dashboard/actions/auto-wire-parameters/actions.ts`